src/components/advanced-color-picker/advanced-color-picker.style.ts

```diff
@@ -11,12 +11,24 @@ import Dialog from "../dialog/dialog.component";
 import StyledIconButton from "../icon-button/icon-button.style";
 import checkerBoardSvg from "../simple-color-picker/simple-color/checker-board.svg";
 import baseTheme from "../../style/themes/base";
+import { Dl } from "../definition-list";
 
 const StyledAdvancedColorPickerWrapper = styled.div`
   ${margin}
   display: inline-block;
 `;
 
+/** To be replaced by accessibly hidden class added in FE-5503 */
+const HiddenCurrentColorList = styled(Dl)`
```

Comment:
I'd prefer to put these styles in a global location (in this PR I have it in `styles/utils` https://github.com/Sage/carbon/pull/5931 ) as we know we need this in a few different places. I'm a little worried that whichever of these PRs gets merged second will need to refactor, and it's unlikely it will be flagged as a conflict on Github and we'll simply end up with the same styles defined in 2 separate places.

Reply:
Agree with Robin here and we will need to be careful we don't merge with same code in two different places.